### DISCUSSION TITLE
请问怎么处理上传进度呢？云盘上传接口看起来没有分片上传的机制？

### DISCUSSION THREAD

**Discussion by wood3n** [Binaryify/NeteaseCloudMusicApi#1282] [Q&A] [answered]
上传以后直接整个文件就扔进去了，接口pending完了最后直接返回结果的

![image](https://user-images.githubusercontent.com/31716713/124150033-c0059000-dac3-11eb-811a-f31a86f3a993.png)

**Comment by Binaryify** [comment, marked as answer]
暂时没这个机制
